server/middleware/api.js

```diff
@@ -55,8 +57,14 @@ export default (store) => (next) => (action) => {
       fetchSpotifyToken(store, action, next);
     },
 
-    [EMIT_REFRESH_SPOTIFY_CODE]() {
-      store.getState().meta.spotifyApi.refreshAccessToken();
+    [EMIT_SPOTIFY_REFRESH_TOKEN_UPDATE]() {
```

Comment:
This was undefined before.
![](https://media.giphy.com/media/wMvESGxZ0Cqd2/giphy.gif)